lua/neogit/config.lua

```diff
@@ -19,159 +19,444 @@ function M.get_reversed_status_maps()
   return result
 end
 
-M.values = {
-  disable_hint = false,
-  disable_context_highlighting = false,
-  disable_signs = false,
-  disable_commit_confirmation = false,
-  disable_builtin_notifications = false,
-  telescope_sorter = function()
-    return nil
-  end,
-  disable_insert_on_commit = true,
-  use_per_project_settings = true,
-  remember_settings = true,
-  use_magit_keybindings = false,
-  auto_refresh = true,
-  sort_branches = "-committerdate",
-  kind = "tab",
-  -- The time after which an output console is shown for slow running commands
-  console_timeout = 2000,
-  -- Automatically show console if a command takes more than console_timeout milliseconds
-  auto_show_console = true,
-  status = {
-    recent_commit_count = 10,
-  },
-  commit_editor = {
-    kind = "split",
-  },
-  commit_select_view = {
+---Returns the default Neogit configuration
+---@return table
+function M.get_default_values()
+  return {
+    disable_hint = false,
+    disable_context_highlighting = false,
+    disable_signs = false,
+    disable_commit_confirmation = false,
+    disable_builtin_notifications = false,
+    telescope_sorter = function()
+      return nil
+    end,
+    disable_insert_on_commit = true,
+    use_per_project_settings = true,
+    remember_settings = true,
+    use_magit_keybindings = false,
```

Comment:
This can be removed, yea?

Reply:
Yeah. Accidentally got included duri g a rebase.

Removing in a moment

Reply:
Oh shit, seems it got included when #659 was merged. Huh.

Reply:
Done